app/models/order.rb

```diff
@@ -17,4 +23,19 @@ class Order < ApplicationRecord
   def purchase_cost
     stock.price_cents * shares
   end
+
+  private
+
+  def create_portfolio_transaction
+    PortfolioTransaction.create!(
```

Comment:
We prob need a follow up ticket here to make sure that the portfolio has sufficient funds before creating the transaction.